docs/spec/openapi.yml

```diff
@@ -481,7 +481,7 @@ paths:
           required: true
           schema:
             type: string
-        - name: name
+        - name: cluster-name
```

Comment:
Doesn't this require some changes in the api package? :thinking:

Reply:
I don't know if it does, I'm searching for it now...

Reply:
I don't think it does for path parameters (only query parameters, I think..)

Reply:
:thinking: 
Doesn't [this function](https://github.com/percona/everest/blob/f0b8dc8f40d3643c71ae1f71ae22587b62815fae/api/database_cluster_backup.go#L49) need to match [this interface](https://github.com/percona/everest/blob/957d3844fc54e8bdd52eec43cdc2c12a60b971a4/api/everest-server.gen.go#L1610)?

Reply:
And it does, the variable names don't have to be the same for it to match 😅

Reply:
Right, it's all good then